cp-quickstart/start-docker-community.sh

```diff
@@ -17,9 +17,9 @@ sleep 2 # give connect an exta moment to fully mature
 echo "connect has started!"
 
 # Configure datagen connectors
-wget -O connector_pageviews_cos.config  https://github.com/confluentinc/kafka-connect-datagen/raw/master/config/connector_pageviews_cos.config
+curl -sS -o connector_pageviews_cos.config  https://github.com/confluentinc/kafka-connect-datagen/raw/master/config/connector_pageviews_cos.config
```

Comment:
append `|| exit 1` here and 2 lines below?

Reply:
👍 done